### PR TITLE
LPS-85557 IE11 select after iframe

### PR DESCRIPTION
**Pull request by SpencerWoo**
https://issues.liferay.com/browse/LPS-85557

Modified from https://github.com/blzaugg/liferay-portal/pull/188 to use `focus()` on one element.  Also used an `if` instead of `else if` to differentiate the functionality of syncWindowsUI and resetFocus.

I couldn't get `blur()` on the closing modal to work.

### CODE REVIEW

File: modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/util_window.js [hunk 1]
```diff
@@ -406,6 +412,14 @@ AUI.add(
 					instance._map[id + instance.IFRAME_SUFFIX] = modal;
 				},
 
+				_resetFocus: function(modal) {
+					var input = AUI(modal).one('input[type=text]');
```
Comment (blzaugg) [inline modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/util_window.js]: @jonmak08, does `AUI(myWidgetInstance)` do something I'm not familiar with?

@SpencerWoo, as far as I can tell `AUI(modal)` does the same as `AUI()`.

I think you want:
```diff
- var input = AUI(modal).one('input[type=text]');
+ var contentBox = modal.get('contentBox');
+ var input = contentBox.one('input[type=text]');
```

File: modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/util_window.js [hunk 2]
```diff
@@ -406,6 +412,14 @@ AUI.add(
...
+
+					if (input) {
+						input._node.focus();
```
Comment (blzaugg) [inline modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/util_window.js]: Change:
```diff
- input._node.focus();
+ input.getDOM().focus();
```

**Comment by blzaugg** [comment]
Send to @JonMak08 after your changes. I'll be out tomorrow (maybe rest of week)

File: modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/util_window.js (hunk 1 above)
Comment (jonmak08) [reply modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/util_window.js]: Hey @blzaugg,

No `AUI(myWidgetInstance)` does not do anything.  `AUI()` is expecting a module argument.

**Pull request closed without merging**